en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,9 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to ``"simple"`` or ``"full"`` to enable versioning 
+   (respectively simple or full level). Set to ``"false"`` to disable versioning
```

Comment:
I just noticed the "full" example below this is also incorrect and, when corrected, would help clarify the syntax.

I can see no precedent in the existing documentation for adding `(string)`. But I think it makes sense to give a type hint going forward.

So I would go for:

````
**versionable**: *(string)* Set to ``simple`` or ``full`` ...
````

Could you also fix the example?